### DISCUSSION TITLE
Colmap - Colab

### DISCUSSION THREAD

**Discussion by pilitro** [colmap/colmap#1466] [Q&A]
I'm executing colmap in google colab (see link below) but at the end, there is an error. Does somebody know how to solve it? (see image)

LINK: https://colab.research.google.com/gist/kwea123/f0e8f38ff2aa94495dbfe7ae9219f75c/colmap_colab.ipynb
![Captura de pantalla 2022-03-16 154826](https://user-images.githubusercontent.com/80844442/158664916-60769032-bf54-4df4-92e8-3febac690d3d.png)

thanks!
kindes!

**Comment by joerowelll** [comment]
colmap is not properly installed, try running the cells before, and dont change the runtime for the whole installation
